Lib/corpuscrawler/crawl_ga.py

```diff
@@ -35,10 +39,48 @@ def _rtenuacht_path(url):
     return rtenuacht or rnagnuacht
 
 
+def _fetch_rte_sitemap(crawler, url, processed=set(), url_filter=lambda x: True):
```

Comment:
Could this go into the regular fetch_sitemap function?

Reply:
As a general purpose change, I'd just keep the mixed namespace/no namespace bit; I think there might be a cleaner way of doing the rest of what I'd been doing here.